src/Admin.php

```diff
@@ -72,11 +72,9 @@ public static function init() {
    * @see http://premium.wpmudev.org/blog/hide-the-wordpress-update-notification/
    */
   public static function removeUselessCoreUpdateNagMessages() {
-    if (!current_user_can('update_core')) {
       // @see wp-admin/includes/admin-filters.php
       remove_action('admin_notices', 'update_nag', 3);
       remove_action('admin_notices', 'maintenance_nag', 10);
-    }
   }
```

Comment:
@Mauricio-Urrego The indentation of the resulting code is still not right - the indentation level is two spaces and the code should not diverge from that:

![image](https://user-images.githubusercontent.com/41992/99403954-58f7ea00-28eb-11eb-9da3-0ff452cfdf51.png)